### PR TITLE
Display client only version during setup

### PR DESCRIPTION
**Pull request by yussufsh**
Signed-off-by: Yussuf Shaikh <yussuf@us.ibm.com>

### CODE REVIEW

File: openshift-install-powervs [hunk 1]
```diff
@@ -1077,20 +1077,15 @@ function setup_ibmcloudcli() {
...
-  OC_PLATFORM="$OS"
   OC_BIN_PREFIX="openshift-client"
   OC_BIN_EXT="tar.gz"
 
-  if [[ "$OS" == "darwin" ]]; then
-    OC_PLATFORM="mac"
-  fi
-
   if [[ "$OS" == "windows" ]]; then
     OC_BIN_EXT="zip"
```
Comment (bpradipt) [inline openshift-install-powervs:1084]: Why not set these variables with OC_CLI_OS during the initial check?

File: openshift-install-powervs [hunk 2]
```diff
@@ -1167,11 +1162,11 @@ function main {
   PLATFORM=$(uname)
   case "$PLATFORM" in
     "Darwin")
-      OS="darwin"; CLI_OS="osx"; PACKAGE_MANAGER="brew"
+      OS="darwin"; IBM_CLI_OS="osx"; OC_CLI_OS="mac"; PACKAGE_MANAGER="brew"
       ;;
     "Linux")
       # Linux distro, e.g "Ubuntu", "RedHatEnterpriseWorkstation", "RedHatEnterpriseServer", "CentOS", "Debian"
-      OS="linux"; CLI_OS="linux64"
+      OS="linux"; IBM_CLI_OS="linux64"; OC_CLI_OS="$OS"
```
Comment (bpradipt) [inline openshift-install-powervs:1169]: How about setting the OC_BIN_EXT variable in the `case` block ?

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by bpradipt** [comment]
need to check with Windows failed

**Comment by yussufsh** [comment]
> 
> 
> need to check with Windows failed

Small change in `unzip` argument.. Fixed it!

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**